api/user.go

```diff
@@ -109,3 +109,29 @@ func (c CvpRestAPI) GetUser(userID string) (*SingleUser, error) {
 	}
 	return &info, nil
 }
+
+// AddUser adds 'user' and returns error if any
+func (c CvpRestAPI) AddUser(user *SingleUser) error {
+	if user == nil {
+		return errors.New("AddUser: can not add nil user")
+	}
+	resp, err := c.client.Post("/user/addUser.do", nil, user)
+	if err != nil {
+		return errors.Errorf("AddUser: %s", err)
+	}
+	var addedUser *SingleUser
```

Comment:
nit: user

Reply:
that variable is supposed to have user data that was returned as response to the api call. 
I named it so, as it reflects the user which was added by cvp into the D.B.